packages/components/src/__actions__/Button/v3/subcomponents/ButtonContent/ButtonContent.module.css

```
@@ -0,0 +1,23 @@
+.buttonContent {
+  display: inline-flex;
+  align-items: center;
+  vertical-align: middle;
+  gap: var(--icon-gap);
```

To match the rename :)
```suggestion
  gap: var(--button-icon-gap);
```